Lib/axisregistry/data/year.textproto

```diff
@@ -0,0 +1,15 @@
+# YEAR based on https://github.com/dancoull/ClimateCrisis
+tag: "YEAR"
+display_name: "Year"
+min_value: -4000
+default_value: 2000
+max_value: 4000
```

Comment:
I understand that these min and max values are for the Climate Crisis font. Wouldn't it be more beneficial to have them as the min and max allowable values of the fixed 16.16 type since this is what font axes use.

> Each axis is defined by a numeric range, using fractional values represented using the Fixed (16.16) data type.

https://learn.microsoft.com/en-us/typography/opentype/spec/otvaroverview#introduction

Reply:
> I understand that these min and max values are for the Climate Crisis font.

The range was actually enlarged from font's original to cover a wider range and with a somehow human-centered relevance.

@m4rc1e you mean using a hexadecimal value? Cause we have defined all the axis in decimal system.